api/gist.js

```diff
@@ -82,6 +83,7 @@ export default async (req, res) => {
         locale: locale ? locale.toLowerCase() : null,
         show_owner: parseBoolean(show_owner),
         hide_border: parseBoolean(hide_border),
+        card_width: parseInt(card_width),
```

Comment:
```suggestion
        card_width: parseInt(card_width, 10),
```

Please add radix to make Eslint happy 😄.